memcached/README.md

```diff
@@ -0,0 +1,45 @@
+# YCSB Memcached binding
+
+## Quick Start
+
+This section describes how to run YCSB on memcached.
+
+### 1. Install and start memcached service on the host(s)
+
+### 2. Install Java and Maven
+
```

Comment:
To make progress on this PR, I've added a link to `../mongodb/README.md` and refer the user to follow step 2 in that doc. Let's consider moving Java / Maven instructions to a common place as a separate PR?